wrappers/services_test.go

```diff
@@ -311,6 +376,48 @@ func (s *servicesTestSuite) TestAddSnapMultiServicesStartFailOnSystemdReloadClea
 	})
 }
 
+func (s *servicesTestSuite) TestAddSnapSocketFiles(c *C) {
+	var sysdLog [][]string
+
+	r := systemd.MockSystemctl(func(cmd ...string) ([]byte, error) {
+		sysdLog = append(sysdLog, cmd)
+		return nil, nil
+	})
+	defer r()
+
+	info := snaptest.MockSnap(c, packageHello+`
+ svc1:
+  daemon: simple
+  sockets:
+    sock1:
+      listen-stream: sock1.socket
+      socket-mode: "0666"
+    sock2:
+      listen-stream: sock2.socket
+`, contentsHello, &snap.SideInfo{Revision: snap.R(12)})
+
+	sock1File := filepath.Join(s.tempdir, "/etc/systemd/system/snap.hello-snap.svc1.sock1.socket")
+	sock2File := filepath.Join(s.tempdir, "/etc/systemd/system/snap.hello-snap.svc1.sock2.socket")
+
+	err := wrappers.AddSnapServices(info, nil)
+	c.Assert(err, IsNil)
+
+	content1, err := ioutil.ReadFile(sock1File)
+	c.Assert(err, IsNil)
+	c.Check(
+		strings.Contains(
+			string(content1),
+			"[Socket]\nService=snap.hello-snap.svc1.service\nListenStream=sock1.socket\nSocketMode=0666\n\n"),
```

Comment:
Is that supported? The manual seems to require a slash as first char to handle it as a path.

Reply:
Ah yeah, that was just to validate template output. I'll change it to be more realistic